### PR TITLE
Finish send mail to admin before 2 days

### PR DESCRIPTION
**Pull request by trantube94**
I have created a pull request, check it please!

### CODE REVIEW

File: app/mailers/user_mailer.rb [hunk 1]
```diff
@@ -3,4 +3,10 @@ def create_account user
     @user = user
     mail to: user.email, subject: I18n.t("mailer.subject")
   end
+
+  def report_admin course, user
+  	@course = course
```
Comment (ttuan) [inline app/mailers/user_mailer.rb]: ident

**Comment by hienbx94** [comment]
Ok

**Comment by ttuan** [comment]
OK

File: app/models/course.rb [hunk 2]
```diff
@@ -31,4 +33,21 @@ def check_status
...
+    end
+  end
+
+  def number_delay_day
+    self.day_work - Settings.delay_days
+  end
+
+  def init_day_work
+    self.day_work = 0
+    self.subjects.each {|subject| self.day_work += subject.day_work}
```
Comment (dieunb) [inline app/models/course.rb]:
```
self.day_work = self.subjects.map(&:day_work).reject(:+)
```

**Pull request merged**